src/containers/User/UserArea.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Button, Container, Label, Segment } from 'semantic-ui-react';
+import { useUserState } from './UserState' 
+
+const UserArea: React.FC = () => {
+    const { 
```

Comment:
I notice the indentation is 4 spaces instead of the 2 specified in `.prettierrc`. So when I save this, it changes back to the more narrow indentation. Perhaps you haven't got your editor set to auto-format on Save? Or something else? 🤷‍♂️

Reply:
Yes. I think mine is still not configured to use. I'll do this now.